### PR TITLE
refactor(shorebird_cli): remove old release commands

### PR DESCRIPTION
**Pull request by bryanoltman**
## Description

Removes the old `ReleaseCommand` now that the rewritten one is in place.

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [x] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore
- [ ] 🧪 Tests

### CODE REVIEW

File: packages/shorebird_cli/lib/src/commands/release/release_command.dart [hunk 1]
```diff
@@ -1,66 +1,448 @@
...
-  }) {
-    logger.err(
-      '''
-${styleBold.wrap(lightRed.wrap('A release with version $releaseVersion already exists but was built using a different Flutter revision.'))}
+  @override
+  Future<int> run() async {
+    final releaserFutures =
+        results.releaseTypes.map(_resolveReleaser).map(createRelease);
+
+    for (final future in releaserFutures) {
```
Comment (erickzanardo) [inline packages/shorebird_cli/lib/src/commands/release/release_command.dart:120]: As I look through I wonder if we should
 - log which release we are doing now
 - that such release worked
 - etc

File: packages/shorebird_cli/lib/src/commands/release/release_command.dart [hunk 2]
```diff
@@ -1,66 +1,448 @@
...
+    await cache.updateAll();
+
+    // This command handles logging, we don't need to provide our own
+    // progress, error logs, etc.
+    final app = await codePushClientWrapper.getApp(appId: appId);
+    final targetFlutterRevision = await resolveTargetFlutterRevision();
+    try {
+      await shorebirdFlutter.installRevision(revision: targetFlutterRevision);
+    } catch (_) {
+      exit(ExitCode.software.code);
```
Comment (erickzanardo) [inline packages/shorebird_cli/lib/src/commands/release/release_command.dart:191]: Should we log this?

**Review by erickzanardo — APPROVED** [review]

File: packages/shorebird_cli/lib/src/commands/release/release_command.dart (hunk 1 above)
Comment (bryanoltman) [reply packages/shorebird_cli/lib/src/commands/release/release_command.dart:120]: The releasers handle the logging, which I think addresses these concerns

File: packages/shorebird_cli/lib/src/commands/release/release_command.dart (hunk 2 above)
Comment (bryanoltman) [reply packages/shorebird_cli/lib/src/commands/release/release_command.dart:191]: shorebirdFlutter logs on its own

**Pull request merged**